usr.sbin/pkg/pkg.7

```diff
@@ -324,7 +324,4 @@ command first appeared in
 It became the default package tool in
 .Fx 10.0 ,
 replacing the
-pkg_install suite of tools
-.Xr pkg_add 1 ,
-.Xr pkg_info 1 and
-.Xr pkg_create 1 .
+pkg_install suite of tools pkg_add(1), pkg_info(1) and pkg_create(1).
```

Comment:
Given this line's HISTORY context in FreeBSD-RELEASE, <https://man.freebsd.org/cgi/man.cgi?query=pkg&sektion=7&manpath=freebsd-release#HISTORY>, I reckon that this change is OK.

Reply:
There is no `pkg_add` page in section 1 though, so I think there's no value in adding the section number suffix.